src/main/java/org/datadog/jmxfetch/AppConfig.java

```diff
@@ -96,6 +96,13 @@ public class AppConfig {
     @Builder.Default
     private boolean logFormatRfc3339 = false;
 
+    @Parameter(
+        names = {"--log_thread_name"},
+        description = "Logs the thread name with each message",
+        required = false)
+    @Builder.Default
+    private boolean logThreadName = true;
```

Comment:
I think this might not work as expected. According to the [docs](https://jcommander.org/#_boolean), to negate a true-by-default boolean parameter, the parameter needs to be configured take the false value explicitly.

But if this is just an escape hatch, then perhaps configuring this via a property would be better. The agent doesn't allow users to set parameters for jmxfetch, so this can not be toggled without an explicit support in the agent. But the agent allows to pass options to the jvm, which can be used to change properties. WDYT?